src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/scanneddocs/ManageScannedDocsContestedAboutToSubmitHandler.java

```diff
@@ -35,7 +39,7 @@ public class ManageScannedDocsContestedAboutToSubmitHandler extends FinremCallba
     public static final String INTERVENER_3 = "Intervener 3 ";
     public static final String INTERVENER_4 = "Intervener 4 ";
 
-    private static List<CaseDocumentType> administrativeCaseDocumentTypes = List.of(
+    private static final List<CaseDocumentType> ADMINISTRATIVE_CASE_DOCUMENT_TYPES = List.of(
```

Comment:
feels like it could be its own enum

Reply:
I'm not sure what we would gain from doing this. Please explain